server/index.js

```diff
@@ -131,6 +113,39 @@ app.get("/api/events", (req, res, next) => {
 
 app.use(authorizationMiddleware);
 
+app.get("/api/eggs", (req, res, next) => {
+  const userId = Number(req.user.id);
+  const anyone = "anyone";
+  const followers = "followers";
+  const allEggQuery = `
+                        select "e".* from "egg" as "e"
+                        where "e"."canClaim" = $1 and "e"."eggId" not in 
+                        (select "f"."eggId" from "foundEggs" as "f")
+                        `;
+  const followerEggQuery = `
+                           select * from "egg"
+                           where "egg"."canClaim" = $1 and "egg"."userId" = (select "followers"."followingId" from "followers" where "followers"."followerId" = $2 and "isAccepted" = true) and "egg"."eggId" not in 
+                           (select "f"."eggId" from "foundEggs" as "f")
+                           `;
+  const allEggParams = [anyone];
+  const followerEggParams = [followers, userId];
+  const eggQueries = [
+    db.query(allEggQuery, allEggParams),
+    db.query(followerEggQuery, followerEggParams),
+  ];
+  const eggPromises = Promise.all(eggQueries);
+  eggPromises
+    .then((result) => {
+      const egg = [...result[0].rows, ...result[1].rows];
+      egg.forEach((egg) => {
```

Comment:
If this is more than one egg, it should be named eggs. Seems minor, but it's low-hanging fruit to get it right. And it makes much more sense. I normally give a break to people who didn't grow up with plural nouns in their first language.